platforms/common/CMakeLists.txt

```diff
@@ -56,6 +56,11 @@ add_library(px4_platform STATIC
 )
 target_link_libraries(px4_platform prebuild_targets px4_work_queue)
 
+if("${PX4_PLATFORM}" MATCHES "qurt")
+# Get rid of fatal warning about "px4_spi_buses != nullptr" comparison in spi.cpp
+set_target_properties(px4_platform PROPERTIES COMPILE_FLAGS -Wno-tautological-pointer-compare)
```

Comment:
Can we address the root cause here instead of scattered platform flags? Is spi.cpp even being compiled on qurt (CONFIG_SPI)? 

https://github.com/PX4/PX4-Autopilot/blob/29ade6c472a2fa83a5c2fe6006902376e88d3cb4/platforms/common/spi.cpp#L36

Reply:
Yes, it's actually quite easy to fix the code to get rid of the error. But you are correct, SPI isn't even part of the build in this PR. I was separating a much larger set of changes into smaller PRs. I'll have to look for a better way to split these up.